slu-scripts/README.md

```diff
@@ -31,72 +32,70 @@ npm run encrypt-file --key_file=./test-data/unclonable.txt --input=./test-data/d
 npm run bootstrap
- --key_file: The key file used to encrypt/decrypt the data 
- --dest: Location where the encrypted data should be saved
- --one-shot-device-url: The url endpoint of the one-shot-device to get the device identity
- --nonce: Nonce of the device
+--key_file: The key file used to encrypt/decrypt the data
+--dest: Location where the encrypted data should be saved
+--one-shot-device-url: The url endpoint of the one-shot-device to get the device identity
+--nonce: Nonce of the device
 ```
 
 Example:
+
 ```
-npm run bootstrap --key_file=./test-data/unclonable.txt --dest=./test-data 
---one-shot-device-url=http://localhost:3000/api/v1/one-shot-device --nonce=46f45a2c-114a-4cea-a570-b379fe3a4a95
+npm run bootstrap --key_file=./test-data/unclonable.txt --dest=./test-data
+--one-shot-device-url=https://cxc.is.iota.cafe/api/v1/one-shot-device/bootstrap --nonce=46f45a2c-114a-4cea-a570-b379fe3a4a95
 ```
 
 <p align="center">
   <img src="https://github.com/iotaledger/cxc-slu-authenticity/blob/69-description-of-slu-scripts/slu-scripts/diagrams/sequence1.png" alt="bootstrap-script sequence diagram"/>
 </p>
 
- 
 3. sending authentication proof:
 
 ```
 npm run send-proof
- --key_file: The key file used to encrypt/decrypt the data 
- --interval: Interval in milliseconds during the proof is send to the authenticity-collecotr microservice. 
+ --key_file: The key file used to encrypt/decrypt the data
+ --interval: Interval in milliseconds during the proof is send to the authenticity-collector microservice.
  --input_enc: The location of the encrypted device identity.
  --collector_base_url: The base url of the authenticity-collector microservice
- ```
+```
+
+Example:
 
- Example:
 ```
 
 npm run send-proof --key_file=./test-data/unclonable.txt --interval=240000 --input_enc=./test-data/data.json.enc
---collector_base_url=http://localhost:3000/api/v1/authenticity 
+--collector_base_url=https://cxc.is.iota.cafe/api/v1/authenticity
 
 ```
+
  <p align="center">
   <img src="https://github.com/iotaledger/cxc-slu-authenticity/blob/69-description-of-slu-scripts/slu-scripts/diagrams/send-proof.png" alt="auth-proof-script sequence diagram"/>
 </p>
- 
 
 4. sending sensor data:
 
 ```
 npm run send-data
  --key_file: The key file used to encrypt/decrypt the data.
  --input_enc: The location of the encrypted device identity.
- --interval: Interval in milliseconds during the data is written into the channel and send to the slu-status microservice. 
+ --interval: Interval in milliseconds during the data is written into the channel and send to the slu-status microservice.
  --collector_base_url: The base url of the authenticity-collector microservice.
  --is_api_key: The api key of the integration services
  --is_base_url: The base url of the integration services
- --jwt: The jwt token of the device to verify itslef by the authenticity-collector microservice.
+ --jwt: The jwt token of the device to verify itself by the authenticity-collector microservice.
```

Comment:
isnt this already removed?

Reply:
@JuriBog

Reply:
yes it was removed in the code in #88. I forgot to remove it from the readme.